packages/studio-web/src/app/upload/upload.component.ts

```diff
@@ -83,7 +84,7 @@ export class UploadComponent implements OnDestroy, OnInit {
     audio: "mic",
     text: "edit",
   };
-  textInput: string = "";
+  textInput = new BehaviorSubject<string>("");
```

Comment:
Since this is an `rxjs` thingy it should have a `$` ... somewhere ... in the name to be consistent.